cartridges/int_turnto_core_v5/cartridge/scripts/util/ServiceFactory.js

```diff
@@ -153,18 +152,16 @@ var ServiceFactory = {
         arrayOfRequestParts.push(new HTTPRequestPart('file', file));
         arrayOfRequestParts.push(new HTTPRequestPart('siteKey', siteKey));
         arrayOfRequestParts.push(new HTTPRequestPart('authKey', authKey));
-        arrayOfRequestParts.push(new HTTPRequestPart('feedStyle', 'tab-style.1'));
+        arrayOfRequestParts.push(new HTTPRequestPart('feedStyle', feedStyle));
```

Comment:
Just for my own understanding.  How are these request parts used?  In other words, what is it calling?

Reply:
This logic was already in place but the `requestDataContainer` is returned with the parts as args. This is passed to FeedUploadService and used in [client.sendMultiPart(requestObject.args)](https://github.com/turnto/salesforce-commerce-cloud/blob/master/cartridges/int_turnto_core_v5/cartridge/scripts/service/FeedUploadService.js#L32)

Reply:
Got it, thanks.  Make more sense to me now.